src/controllers/posts.js

```diff
@@ -10,5 +10,23 @@ exports.get = (req, res) => {
 }
 
 exports.post = (req, res) => {
-  queries.addNewPost()
+  let id = req.body.id;
+  let title = req.body.title;
+  let desc = req.body.desc;
+  queries.addNewPost(id,title,desc,likes,(err,result)=>{
```

Comment:
remove the likes argument

Reply:
Also remove it from the SQL query it self